Sources/Logto/Types/IdTokenClaims.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  IdTokenClaims.swift
+//  
+//
+//  Created by Gao Sun on 2022/1/7.
+//
+
+import Foundation
+
+public struct IdTokenClaims {
+    let sub: String
+    let atHash: String
+    let aud: String
+    let exp: Int64
+    let iat: Int64
+    let iss: String
```

Comment:
These properties are `internal` by default, therefore user cannot get properties of `IdTokenClaims` outside the module. E.g., `print(tokerClaims.sub) // error, the property is internal`.

Not sure if this is by design, if we don't want to expose those properties, just leave this comment :)